src/Symfony/Component/Routing/Loader/AttributeFileLoader.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Loader;
+
+/**
+ * AttributeFileLoader loads routing information from attributes set
+ * on a PHP class and its methods.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class AttributeFileLoader extends AnnotationFileLoader
```

Comment:
I think the other way around would be better if possible: old extends new. The reason is that this will make the code in 6.4 as close as possible to the one we anticipate in 7.0. Please consider this swap for all classes.

Reply:
Alright, I did the swap for all three classes, added `trigger_deprecation` and updated `.github/expected-missing-return-types.diff`